plugins/modules/site_workflow_manager.py

```diff
@@ -760,14 +1025,263 @@ def get_want(self, config):
             parameters such as 'site_params' and 'site_name.' The gathered
             information is stored in the 'want' attribute for later reference.
         """
+        try:
+
+            want_list = []
```

Comment:
You can check the dnac version here itself and return ..

Reply:
Modified